azurerm/resource_arm_api_management.go

```diff
@@ -581,7 +582,7 @@ func resourceArmApiManagementServiceCreateUpdate(d *schema.ResourceData, meta in
 
 		// then add the new one, if it exists
 		if policy != nil {
-			if _, err := policyClient.CreateOrUpdate(ctx, resourceGroup, name, *policy); err != nil {
+			if _, err := policyClient.CreateOrUpdate(ctx, resourceGroup, name, *policy, ""); err != nil { // TODO: Check how to deal with etag
```

Comment:
Need help verifying how to work with the If-Match header / Etags in this plugin.